components/Banner.tsx

```diff
@@ -66,14 +68,21 @@ const Banner = ({ bannerHeight, setBannerHeight }: {
 
 	useEffect(() => {
 		const determineSituation = (): BannerSituation => {
+            const isMobile = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
```

Comment:
How are you sure that these are all the alternatives for detecting whether it is a mobile device?

(Very low LOGAF.)

Reply:
It doesn't (especially for TVs - where Samsung and LG's OSes are more popular than Google TV or Android OS). 
But it is alright. We will deal with it later.